CHANGELOG.md

```diff
@@ -4,6 +4,46 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.0.0
+
+*October 24, 2023*
+
+This is the official v1.0.0 release of NGINX Gateway Fabric.
+
+BREAKING CHANGES:
+
+- Rename the product from NGINX Kubernetes Gateway to NGINX Gateway Fabric. [PR-1070](https://github.com/nginxinc/nginx-gateway-fabric/pull/1070)
+
+FEATURES:
+
+- Add readiness probe. [PR-1047](https://github.com/nginxinc/nginx-gateway-fabric/pull/1047)
+- Support horizontal scaling. [PR-1048](https://github.com/nginxinc/nginx-gateway-fabric/pull/1048)
+- Add NGINX reload counters. [PR-1049](https://github.com/nginxinc/nginx-gateway-fabric/pull/1049)
```

Comment:
Should we mention the event batch processing metric?

Reply:
The reload metrics are more than just a counter. Maybe something more generic like: 
```suggestion
- Add NGINX reload metrics. [PR-1049](https://github.com/nginxinc/nginx-gateway-fabric/pull/1049)
```